opensourceleg/hardware/sensors.py

```diff
@@ -223,14 +223,14 @@ def initialize(self, number_of_iterations: int = 2000) -> None:
             self._zeroed = True
             self._log.info(f"[{self.__repr__()}] Zeroing routine complete.")
 
-        elif (
-            input(
-                f"[{self.__repr__()}] Would you like to re-initialize loadcell? (y/n): "
-            )
-            == "y"
-        ):
+        elif reset:
```

Comment:
While we're here, we should add an input wait before zeroing starts so the user can confirm it is unloaded. Otherwise it can kinda sneak up on you. I'm thinking replacing the `time.sleep(1)` on line 202 with it.

Reply:
[65ec987](https://github.com/neurobionics/opensourceleg/pull/236/commits/65ec987f4d30ada9143e6a20a3f8d74fef6858e2)

Reply:
Yes! Here is one under the `make_encoder_map` method on our `main` branch for reference: https://github.com/neurobionics/opensourceleg/blob/d2bf41593cf72e0c94799511143528506b83ca85/opensourceleg/hardware/joints.py#L196

Reply:
Cool, I trust you.